test/System/Angle.test.cpp

```diff
@@ -202,27 +202,27 @@ TEST_CASE("[System] sf::Angle")
         {
             sf::Angle angle = sf::degrees(15);
             angle -= sf::degrees(15);
-            CHECK(angle == sf::degrees(0));
+            CHECK(angle == Approx(sf::degrees(0)));
             angle -= sf::radians(10);
             CHECK(angle == sf::radians(-10));
         }
 
         SECTION("operator*")
         {
             STATIC_CHECK(sf::radians(0) * 10 == sf::Angle::Zero);
-            STATIC_CHECK(sf::degrees(10) * 2.5f == sf::degrees(25));
-            STATIC_CHECK(sf::degrees(100) * 10.0f == sf::degrees(1000));
+            CHECK(sf::degrees(10) * 2.5f == Approx(sf::degrees(25)));
+            CHECK(sf::degrees(100) * 10.0f == Approx(sf::degrees(1000)));
 
             STATIC_CHECK(10 * sf::radians(0) == sf::Angle::Zero);
-            STATIC_CHECK(2.5f * sf::degrees(10) == sf::degrees(25));
-            STATIC_CHECK(10.0f * sf::degrees(100) == sf::degrees(1000));
+            CHECK(2.5f * sf::degrees(10) == Approx(sf::degrees(25)));
+            CHECK(10.0f * sf::degrees(100) == Approx(sf::degrees(1000)));
```

Comment:
Anywhere we previously did exact comparison with degrees I'd rather we do exact comparisons with radians rather than start doing approximate comparisons.

Reply:
Just wanted to put in my two cents because I disagree with this point.

In my opinion, all of these tests should do approx comparisons either way as with any other float test.

By testing for exact values of float, you are no longer only testing if the API works but also testing for implementation details.